tests/reftests/pin.test

```diff
@@ -175,25 +996,101 @@ echo "another-inexistant" "inexistant"
 -> installed qux.dev
 -> installed bar.dev
 Done.
-### : Test opam pin remove <pkg>.<version>
-### opam pin remove --no-action bar.dev
-Ok, bar is no longer pinned to file://${BASEDIR}/bar (version dev)
+### :XI:c: Test opam pin remove <pkg>.<version>
```

Comment:
Should this test & remove --all be in its proper section above ?

Reply:
i think it's fine here